Add EU PoA and update EUCC rulebooks (#55)
webuild-consortium/webuild-attestation-rulebooks-catalog: rulebooks/ds004-eucc-rulebook.md

Reviewer: This point goes to my comment:  is this attestation a QEAA, pubEAA or simple EAA?  QEAAs MUST be issued by a QTSP, pubEAAs by a certified body, and EAAs by anyone who wants to build an issuer. 

I would change this line to suggested change. 

Please address this in both rulebooks.

Reviewer: ```suggestion
* The Authentic Source of the data contained in the EUCC MUST be the company register.   The company register can or MUST authorize an issuer to issue EUCCs on their behalf. 
```

Author: Added

* Author(s):
    <!-- Usage help: (Name LastName, Affiliation) -->
  * Jonas Toennis, [Brønnøysund Registry Center](https://www.brreg.no)
  * Erwin Nieuwlaar, [KVK](https://www.kvk.nl)

> 

Reviewer: Should Astrid also be added here?

Author: Good point! Added.

| 02                | 10.04.2026         | Updated according to the work done in WE BUILD, using the [WE BUILD EUCC attestation description](https://portal.webuildconsortium.eu/group/bu2-create-company-branch/files?mid=7087&fid%5B0%5D=56&fid%5B1%5D=58&fid%5B2%5D=62) |

**Feedback:**
Main feedback channel: [GitHub issues](https://github.com/webuild-consortium/eudi-wallet-rulebooks-and-schemas/issues)
Alternative: Contact Business usecase 2 contact points in WE BUILD.

## 1 Introduction

### 1.1 <mark style="background-color: yellow">Attestation introduction</mark>

This document is the EU Company Certificate Data Rulebook (referred to as EUCC) based on
the [EU Company Law regulation](https://eur-lex.europa.eu/eli/dir/2025/25/oj/eng). It contains the specific requirements, issuance process, formatting and content of the EUCC. 
This document is created and used by the WEBUILD consortium to provide a common understanding, data schema, format and usage guidlines for the WEBUILD eco system. 

> This document MAY be used as one of multiple innputs toward a final EUCC definition for the EBW ecosystem but that is up so the sole decisision of the EU commission or any other relevant authority with delgated power from the commission.

Reviewer: ```suggestion
This document MAY be used as one of multiple innputs toward a final EUCC definition for the EBW ecosystem. That is under the sole discreation of the EU commission or any other relevant authority with delgated power from the commission.
```

whereas other words, such as 'will', and 'is' or 'are' are intended as
statements of fact.

### 1.<mark style="background-color: yellow">5</mark> Terminology

> This document uses terminology specified in [Regulation (EU) 2024/1183](https://eur-lex.europa.eu/eli/reg/2024/1183/oj/eng).

Reviewer 2: From the the template: "It is recommended to use the terminology defined in Annex 1 of ARF."  I suggest to include also "and Annex 1 of the ARF https://eudi.dev/1.4.0/annexes/annex-1/annex-1-definitions/" as there are references to pub-EAA and other terms defined there.

Author: Added!
